packages/pages-components/src/map/providerPin.ts

```diff
@@ -19,6 +19,7 @@ class ProviderPinOptions {
   hoverHandler: PinHoverHandler;
   icons: { [key: string]: string };
   hasPinUrl: boolean;
+  instance: typeof mapboxgl;
```

Comment:
Isn't the provider pin agnostic of the provider used? Now it's locked into mapbox.